packages/aws-cdk/test/api/hotswap/fake-printer.ts

```diff
@@ -0,0 +1,16 @@
+import { IEventPrinter, CloudWatchLogEvent } from '../../../lib/api/monitor/logs-monitor';
```

Comment:
This file is probably in the wrong directory now. `test/api/logs/` would be my choice.